### PR TITLE
Added some useful resources from my robotics experience.

### PR DESCRIPTION
**Pull request by chris24sahadeo**

### CODE REVIEW

**Comment by chris24sahadeo** [comment]
@msadowski Requesting review & approval :D

File: README.md [hunk 1]
```diff
@@ -69,6 +70,9 @@ A collection of useful links discovered through the work on [Weekly Robotics](ht
 
 
 ## Libraries and Frameworks
+ * [Rerun](rerun.io) - An SDK for logging computer vision and robotics data paired with a visualizer for exploring that data over time. It lets you debug and understand the internal state and data of your systems with minimal code.
```
Comment (msadowski) [inline README.md:73]: Usually, I don't feature companies here. Could you instead link to their repo https://github.com/rerun-io/rerun ? For each of the entries. Could you append `Licence: {{PROJECT_LICENCE}}` for each of the projects? It's probably tricky for Turtlebot but the rest should have quite clear licensing I hope

**Review by msadowski — COMMENTED** [review]
Thanks for the contribution and tagging me so I could see the PR! I added some comments. Appreciate the help!

File: README.md (hunk 1 above)
Comment (chris24sahadeo) [reply README.md:73]: Noted on the "no companies" and ***facepalm*** totally missed the Licences. Done!

**Review by msadowski — APPROVED** [review]
Awesome, thanks a lot!!

**Pull request merged**